packages/dev/core/src/Meshes/GaussianSplatting/gaussianSplattingMesh.ts

```diff
@@ -92,6 +93,8 @@ export class GaussianSplattingMesh extends Mesh {
         if (url) {
             this.loadFileAsync(url);
         }
+        this._material = new GaussianSplattingMaterial(this.name + "_material", this._scene);

```

Comment:
Why not directly assigning to `this.material`?

Reply:
Left over from previous changes, I guess. Let me try to remove _material.